ConwaysGameOfLife.Tests/GeneratorShould.cs

```diff
@@ -10,8 +10,9 @@ public GeneratorShould()
         {
             _generator = new Generator(new Universe(3, 3));
         }
+
         [Fact]
-        public void WhenInitialized_DisplayUniverse()
+        public void WhenUniverseCreatedInitially_DisplayUniverse() // TODO: if parameter on line 19 is removed, test fails

```

Comment:
+1 thanks for clarifying

probably need to inject the `StubOutput` as part of the setup for this test once `DisplayUniverse` no longer takes the `IOutput` as a param